### ISSUE TITLE
Allow for more control over profile folder path

### ISSUE THREAD

**Issue by FatalUserK** [modrinth/theseus#1132]
### Is your suggested enhancement related to a problem? Please describe.

While I do feel it can be nice to separate your modded profiles, I feel you should have more control over the folder path it goes to.

Specifically I have a client-side profile that has some various optimisation and client-side QoL mods installed, but if I want to access things from my .minecraft, such worlds, server lists, or saved hotbars, I need to manually copy files over to the specific profile I am playing on

Also redoing vanilla settings every time is a pain in the ass, would be nice to keep it universal

### Describe the solution you'd like

The ability to manually designate a folder-path for your profile in your profile's settings, most other minecraft launchers, including the official one, gives you freedom over where your minecraft is run from

If there are concerns about multiple profiles manually being set to a single location, you could apply warnings, or even have the profile check if any other profiles lead there and let the player know this break some mods

### Describe alternatives you've considered

An alternative would be a menu that allows you to transfer certain things from different profiles and even vanilla profiles

Things you could include:

- Worlds tab that allows you to select worlds you would like to copy over (and if you would like to delete them from where you copied them or maybe make a back up), or have options to copy over the server list file
- hotbars file, 
- the vanilla options 
- or maybe even certain modded configs you have set up

### Additional context

just seems kind of weird how there are overrides for java and environmental variable and args but not an override for the folder path, would be very much appreciated 👍

**Comment by fernandossmm** [comment]
Yeah, an option for the "Game Directory", like in the official launcher, would be great. Let the user override default installation path if desired. Honestly, seems like a basic feature.

The transfer options seems to me a bit extreme, kind of "forcing" users to stick with modrinth; and duplicating worlds is not something I'd recommend.

**Comment by Prospector** [comment]
We have plans to
